apps/jubmoji-quest/src/components/LeaderBoard.tsx

```diff
@@ -37,37 +68,45 @@ export default function LeaderBoard() {
           <RankLabel variant="primary">Name</RankLabel>
           <RankLabel variant="primary">Score</RankLabel>
         </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel variant="active">Rank</RankLabel>
-          <RankLabel variant="active">Name</RankLabel>
-          <RankLabel variant="active">Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
+        {loading ? (
+          <>
+            <LoadingContent />
+            <LoadingContent />
+            <LoadingContent />
+            <LoadingContent />
+          </>
+        ) : hasRanking ? (
+          <div className="overflow-scroll max-h-40 pr-1">
+            <div className="flex flex-col gap-2 w-full">
+              {ranking?.map(([pubKeyIndex, score], index) => {
+                const rank = index + 1;
+                const card = getJubmojiCardByPubIndex(
+                  jubmojiCollectionCards,
+                  Number(pubKeyIndex)
+                );
+
+                return (
```

Comment:
Can you mock what it would look like when a user is on a specific team? Would their leaderboard row be highlighted? Actually, you might just be able to implement this, filter a user's Jubmojis and see if they have one which corresponds to a team in the leaderboard

Reply:
Here we go, should be highlighted as this when is part of the team: 

<img  src="https://github.com/jubmoji/jubmoji.quest/assets/20865711/1e5a80cc-aa81-49ac-b8b0-d4a6b88b7368" width="300px">

Reply:
looks great!